src/applications/facility-locator/components/search-results-items/UrgentCareResult.jsx

```diff
@@ -48,9 +47,6 @@ const UrgentCareResult = ({ provider, query }) => {
           <a
             href="https://www.va.gov/COMMUNITYCARE/programs/veterans/Urgent-Care.asp"
             target="_/blank"
-            onClick={() => {
```

Comment:
This event is duplicative (and not informative?) and the event from the `va-alert` is sufficient for our purposes.

<img width="700" alt="Screenshot 2024-08-08 at 3 06 46 PM" src="https://github.com/user-attachments/assets/2051b940-1ecc-4ffe-9f95-039ee581b54b">